lib/puppet_x/binford2k/node_encrypt.rb

```diff
@@ -81,7 +81,7 @@ class Value
         attr_accessor :decrypted_value
 
         def initialize(value)
-          if Puppet_X::Binford2k::NodeEncrypt.encrypted? value
+          if Puppet_X::Binford2k::NodeEncrypt.encrypted? value and not defined?($onceover_node)
```

Comment:
can we skip this if the current environment is `rp_env` (the environment set by rspec-puppet)?  That would cover both onceover and rspec-puppet more generally.

Reply:
If that sounds ok, I'll try it out and update my PR.